service/src/main/java/bio/terra/externalcreds/dataAccess/GA4GHPassportDAO.java

```diff
@@ -18,9 +18,12 @@
 public class GA4GHPassportDAO {
 
   final NamedParameterJdbcTemplate jdbcTemplate;
+  final LinkedAccountDAO linkedAccountDAO;
```

Comment:
don't need this and in general daos should not depend on other daos

Reply:
oh oops that was left over from a previous thing I was trying, thanks!